eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/Gateway.php

```diff
@@ -244,4 +244,25 @@
      * @param int[] $languageIds Language IDs of removed Translations
      */
     abstract public function archiveUrlAliasesForDeletedTranslations($locationId, $parentId, array $languageIds);
+
+    /**
+     * Delete URL aliases pointing to non-existent Locations.
+     *
+     * @return int Number of affected rows.
+     */
+    abstract public function deleteUrlAliasesWithoutLocation();
```

Comment:
Can we also add return type hints for these three methods?

Reply:
Fixed via 51ae1c2